scheds/rust/scx_layered/src/bpf/intf.h

```diff
@@ -249,6 +250,7 @@ struct layer_match {
 	u32		ppid;
 	u32		tgid;
 	u64		nsid;
+	u32		pid_tgid_eq;
```

Comment:
should this be a bool?

Reply:
bool's a macro not a type, the ffi w/ this looked weird.

Reply:
I'd prefer `bool` too. How is ffi weird?

Reply:
It picks a funny type for bool (`MaybeUninit<bool>`, vs just the primitive type for `u32`). That being said playing with it a bit, it works as well as `u32`. Changed to bool.